themes/custom/droopler_theme/scss/components/_d_p_side_by_side.scss

```diff
@@ -16,6 +16,7 @@ $d-p-side-by-side-padding-y: 5% !default;
 $d-p-side-by-side-subtitle-margin-y: 2.125rem !default;
 
 $d-p-side-by-side-content-padding-x: 7% !default;
+$d-p-side-by-side-content-padding-x-lg-item-1: 0 $d-p-side-by-side-content-padding-x 0 $grid-gutter-width !default;
```

Comment:
Nazwałbym tę zmienną `$d-p-side-by-side-content-padding-lg-item1`, bo to nie jest padding poziomy tylko całościowy
Czy to będzie działało jeśli cena będzie w prawym elemencie paragrafu, a w lewym nie?